mathinspector/__init__.py

```diff
@@ -157,7 +158,58 @@ def __init__(self, *args, debug=False, disable=[]):
 			self.modules.addfile(i)
 
 
-def main(*args, **kwargs):
+def main(argv=None):
```

Comment:
Given that the cli for mathinspector can potentially have a wide range of options which do not require the app to launch, I am concerned that placing the main function in `__init__` violates the principle that a single file should a have a single dedicated purpose.

In this case, since both the import and cli interfaces have plans for major expansions, keeping the cli in `__init__` will harm the readability of the code, the longer and more these two interfaces become and the more independent functionality they have.

Reply:
Wouldn't it be better to call this function `parseargv` from a code readability standpoint?  I find the concept that a file named `__main__` calls a function named `main` in a file named `__init__` whose purpose is to parse the argv's and pass them to another function called `run()` which does what typical `main()` functions do, very confusing!